### ISSUE TITLE
[boost, pcl, cgal] build failure

### ISSUE THREAD

**Issue by chatyan** [microsoft/vcpkg#12045]
**Host Environment**
- OS: [Windows 10, x86]

**To Reproduce**
`./vcpkg install pcl and cgal

**Failure logs** 
.\vcpkg install cgal
Computing installation plan...
The following packages will be built and installed:
  * boost-algorithm[core]:x86-windows
  * boost-any[core]:x86-windows
  * boost-array[core]:x86-windows
  * boost-assert[core]:x86-windows
  * boost-atomic[core]:x86-windows
  * boost-bimap[core]:x86-windows
  * boost-bind[core]:x86-windows
  * boost-build[core]:x86-windows
  * boost-chrono[core]:x86-windows
  * boost-circular-buffer[core]:x86-windows
  * boost-compatibility[core]:x86-windows
  * boost-concept-check[core]:x86-windows
  * boost-config[core]:x86-windows
  * boost-container[core]:x86-windows
  * boost-container-hash[core]:x86-windows
  * boost-conversion[core]:x86-windows
  * boost-core[core]:x86-windows
  * boost-date-time[core]:x86-windows
  * boost-detail[core]:x86-windows
  * boost-dynamic-bitset[core]:x86-windows
  * boost-endian[core]:x86-windows
  * boost-exception[core]:x86-windows
  * boost-filesystem[core]:x86-windows
  * boost-foreach[core]:x86-windows
  * boost-format[core]:x86-windows
  * boost-function[core]:x86-windows
  * boost-function-types[core]:x86-windows
  * boost-functional[core]:x86-windows
  * boost-fusion[core]:x86-windows
  * boost-graph[core]:x86-windows
  * boost-heap[core]:x86-windows
  * boost-integer[core]:x86-windows
  * boost-interval[core]:x86-windows
  * boost-intrusive[core]:x86-windows
  * boost-io[core]:x86-windows
  * boost-iostreams[core]:x86-windows
  * boost-iterator[core]:x86-windows
  * boost-lambda[core]:x86-windows
  * boost-lexical-cast[core]:x86-windows
  * boost-locale[core]:x86-windows
  * boost-logic[core]:x86-windows
  * boost-math[core]:x86-windows
  * boost-modular-build-helper[core]:x86-windows
  * boost-move[core]:x86-windows
  * boost-mp11[core]:x86-windows
  * boost-mpl[core]:x86-windows
  * boost-multi-index[core]:x86-windows
  * boost-multiprecision[core]:x86-windows
  * boost-numeric-conversion[core]:x86-windows
  * boost-optional[core]:x86-windows
  * boost-parameter[core]:x86-windows
  * boost-phoenix[core]:x86-windows
  * boost-pool[core]:x86-windows
  * boost-predef[core]:x86-windows
  * boost-preprocessor[core]:x86-windows
  * boost-property-map[core]:x86-windows
  * boost-property-tree[core]:x86-windows
  * boost-proto[core]:x86-windows
  * boost-ptr-container[core]:x86-windows
  * boost-random[core]:x86-windows
  * boost-range[core]:x86-windows
  * boost-ratio[core]:x86-windows
  * boost-rational[core]:x86-windows
  * boost-regex[core]:x86-windows
  * boost-serialization[core]:x86-windows
  * boost-smart-ptr[core]:x86-windows
  * boost-spirit[core]:x86-windows
  * boost-static-assert[core]:x86-windows
  * boost-system[core]:x86-windows
  * boost-test[core]:x86-windows
  * boost-thread[core]:x86-windows
  * boost-throw-exception[core]:x86-windows
  * boost-tokenizer[core]:x86-windows
  * boost-tti[core]:x86-windows
  * boost-tuple[core]:x86-windows
  * boost-type-index[core]:x86-windows
  * boost-type-traits[core]:x86-windows
  * boost-typeof[core]:x86-windows
  * boost-unordered[core]:x86-windows
  * boost-utility[core]:x86-windows
  * boost-variant[core]:x86-windows
  * boost-winapi[core]:x86-windows
  * boost-xpressive[core]:x86-windows
  * bzip2[core]:x86-windows
    cgal[core]:x86-windows
  * liblzma[core]:x86-windows
  * mpfr[core]:x86-windows
  * mpir[core]:x86-windows
  * zlib[core]:x86-windows
  * zstd[core]:x86-windows
Additional packages (*) will be modified to complete this operation.
Starting package 1/90: boost-compatibility:x86-windows
Building package boost-compatibility[core]:x86-windows...
-- Using cached D:/Work 2019/GIT Code/vcpkg/downloads/boostorg-compatibility-boost-1.73.0.tar.gz
-- Using source at D:/Work 2019/GIT Code/vcpkg/buildtrees/boost-compatibility/src/ost-1.73.0-44642da97c
-- Packaging headers
-- Packaging headers done
-- Using cached D:/Work 2019/GIT Code/vcpkg/downloads/boost_LICENSE_1_0.txt
CMake Error at scripts/cmake/vcpkg_download_distfile.cmake:99 (message):

  File does not have expected hash:

          File path: [ D:/Work 2019/GIT Code/vcpkg/downloads/boost_LICENSE_1_0.txt ]
      Expected hash: [ d6078467835dba8932314c1c1e945569a64b065474d7aced27c9a7acc391d52e9f234138ed9f1aa9cd576f25f12f557e0b733c14891d42c16ecdc4a7bd4d60b8 ]
        Actual hash: [ b3e1ca08971b1c3e8cb11a854d094cea3a5fd2c95cf9e985b5212a4bde7975cc3ae15455e8d9e7dc3f2a47b1285b0a34798fc6e194d4ec2db4ceb76547bf07bb ]

  Please delete the file and retry if this file should be downloaded again.

Call Stack (most recent call first):
  scripts/cmake/vcpkg_download_distfile.cmake:110 (test_hash)
  installed/x86-windows/share/boost-vcpkg-helpers/boost-modular-headers.cmake:17 (vcpkg_download_distfile)
  ports/boost-compatibility/portfile.cmake:12 (boost_modular_headers)
  scripts/ports.cmake:76 (include)

Error: Building package boost-compatibility:x86-windows failed with: BUILD_FAILED
Please ensure you're using the latest portfiles with `.\vcpkg update`, then
submit an issue at https://github.com/Microsoft/vcpkg/issues including:
  Package: boost-compatibility:x86-windows
  Vcpkg version: 2020.02.04-nohash

Additionally, attach any relevant sections from the log files above.

**Comment by NancyLi1013** [comment]
Hi @chatyan 
Thanks for posting this issue.
Could you please remove  _D:/Work 2019/GIT Code/vcpkg/downloads/boost_LICENSE_1_0.txt_ and rebuild this?

**Comment by chatyan** [comment]
@NancyLi1013 HI. when I remove the license file, it shows another problem. 
D:\Work 2019\GIT Code\vcpkg>.\vcpkg install pcl
Computing installation plan...
The following packages will be built and installed:
  * boost-algorithm[core]:x86-windows
  * boost-any[core]:x86-windows
  * boost-array[core]:x86-windows
  * boost-asio[core]:x86-windows
  * boost-assert[core]:x86-windows
  * boost-atomic[core]:x86-windows
  * boost-bimap[core]:x86-windows
  * boost-bind[core]:x86-windows
  * boost-build[core]:x86-windows
  * boost-chrono[core]:x86-windows
  * boost-circular-buffer[core]:x86-windows
  * boost-compatibility[core]:x86-windows
  * boost-concept-check[core]:x86-windows
  * boost-config[core]:x86-windows
  * boost-container[core]:x86-windows
  * boost-container-hash[core]:x86-windows
  * boost-context[core]:x86-windows
  * boost-conversion[core]:x86-windows
  * boost-core[core]:x86-windows
  * boost-coroutine[core]:x86-windows
  * boost-date-time[core]:x86-windows
  * boost-detail[core]:x86-windows
  * boost-dynamic-bitset[core]:x86-windows
  * boost-endian[core]:x86-windows
  * boost-exception[core]:x86-windows
  * boost-filesystem[core]:x86-windows
  * boost-foreach[core]:x86-windows
  * boost-format[core]:x86-windows
  * boost-function[core]:x86-windows
  * boost-function-types[core]:x86-windows
  * boost-functional[core]:x86-windows
  * boost-fusion[core]:x86-windows
  * boost-graph[core]:x86-windows
  * boost-integer[core]:x86-windows
  * boost-interprocess[core]:x86-windows
  * boost-intrusive[core]:x86-windows
  * boost-io[core]:x86-windows
  * boost-iostreams[core]:x86-windows
  * boost-iterator[core]:x86-windows
  * boost-lambda[core]:x86-windows
  * boost-lexical-cast[core]:x86-windows
  * boost-locale[core]:x86-windows
  * boost-math[core]:x86-windows
  * boost-modular-build-helper[core]:x86-windows
  * boost-move[core]:x86-windows
  * boost-mp11[core]:x86-windows
  * boost-mpl[core]:x86-windows
  * boost-multi-array[core]:x86-windows
  * boost-multi-index[core]:x86-windows
  * boost-multiprecision[core]:x86-windows
  * boost-numeric-conversion[core]:x86-windows
  * boost-optional[core]:x86-windows
  * boost-parameter[core]:x86-windows
  * boost-phoenix[core]:x86-windows
  * boost-pool[core]:x86-windows
  * boost-predef[core]:x86-windows
  * boost-preprocessor[core]:x86-windows
  * boost-property-map[core]:x86-windows
  * boost-property-tree[core]:x86-windows
  * boost-proto[core]:x86-windows
  * boost-ptr-container[core]:x86-windows
  * boost-random[core]:x86-windows
  * boost-range[core]:x86-windows
  * boost-ratio[core]:x86-windows
  * boost-rational[core]:x86-windows
  * boost-regex[core]:x86-windows
  * boost-serialization[core]:x86-windows
  * boost-signals2[core]:x86-windows
  * boost-smart-ptr[core]:x86-windows
  * boost-spirit[core]:x86-windows
  * boost-static-assert[core]:x86-windows
  * boost-system[core]:x86-windows
  * boost-test[core]:x86-windows
  * boost-thread[core]:x86-windows
  * boost-throw-exception[core]:x86-windows
  * boost-tokenizer[core]:x86-windows
  * boost-tti[core]:x86-windows
  * boost-tuple[core]:x86-windows
  * boost-type-index[core]:x86-windows
  * boost-type-traits[core]:x86-windows
  * boost-typeof[core]:x86-windows
  * boost-unordered[core]:x86-windows
  * boost-utility[core]:x86-windows
  * boost-uuid[core]:x86-windows
  * boost-variant[core]:x86-windows
  * boost-winapi[core]:x86-windows
  * boost-xpressive[core]:x86-windows
  * bzip2[core]:x86-windows
  * curl[core,non-http,ssl,sspi,winssl]:x86-windows
  * double-conversion[core]:x86-windows
  * expat[core]:x86-windows
  * flann[core]:x86-windows
  * freetype[bzip2,core,png]:x86-windows
  * glew[core]:x86-windows
  * hdf5[core,szip,zlib]:x86-windows
  * jsoncpp[core]:x86-windows
  * libharu[core,notiffsymbols]:x86-windows
  * libiconv[core]:x86-windows
  * libjpeg-turbo[core]:x86-windows
  * liblzma[core]:x86-windows
  * libogg[core]:x86-windows
  * libpng[core]:x86-windows
  * libtheora[core]:x86-windows
  * libxml2[core]:x86-windows
  * lz4[core]:x86-windows
  * netcdf-c[core]:x86-windows
  * openssl[core]:x86-windows
  * openssl-windows[core]:x86-windows
    pcl[core]:x86-windows
  * pegtl-2[core]:x86-windows
  * proj4[core,database]:x86-windows
  * pugixml[core]:x86-windows
  * qhull[core]:x86-windows
  * sqlite3[core,tool]:x86-windows
  * szip[core]:x86-windows
  * tiff[core]:x86-windows
  * utfcpp[core]:x86-windows
  * vtk[core]:x86-windows
  * xxhash[core]:x86-windows
  * zlib[core]:x86-windows
  * zstd[core]:x86-windows
Additional packages (*) will be modified to complete this operation.
Starting package 1/121: boost-compatibility:x86-windows
Building package boost-compatibility[core]:x86-windows...
-- Using cached D:/Work 2019/GIT Code/vcpkg/downloads/boostorg-compatibility-boost-1.73.0.tar.gz
-- Using source at D:/Work 2019/GIT Code/vcpkg/buildtrees/boost-compatibility/src/ost-1.73.0-44642da97c
-- Packaging headers
-- Packaging headers done
-- Downloading https://raw.githubusercontent.com/boostorg/boost/boost-1.73.0/LICENSE_1_0.txt...
-- Downloading https://raw.githubusercontent.com/boostorg/boost/boost-1.73.0/LICENSE_1_0.txt... Failed. Status: 6;"Couldn't resolve host name"
CMake Error at scripts/cmake/vcpkg_download_distfile.cmake:175 (message):

      Failed to download file.
      If you use a proxy, please set the HTTPS_PROXY and HTTP_PROXY environment
      variables to "https://user:password@your-proxy-ip-address:port/".
      Otherwise, please submit an issue at https://github.com/Microsoft/vcpkg/issues

Call Stack (most recent call first):
  installed/x86-windows/share/boost-vcpkg-helpers/boost-modular-headers.cmake:17 (vcpkg_download_distfile)
  ports/boost-compatibility/portfile.cmake:12 (boost_modular_headers)
  scripts/ports.cmake:76 (include)


Error: Building package boost-compatibility:x86-windows failed with: BUILD_FAILED
Please ensure you're using the latest portfiles with `.\vcpkg update`, then
submit an issue at https://github.com/Microsoft/vcpkg/issues including:
  Package: boost-compatibility:x86-windows
  Vcpkg version: 2020.02.04-nohash

Additionally, attach any relevant sections from the log files above.

**Comment by NancyLi1013** [comment]
Could you please download `https://raw.githubusercontent.com/boostorg/boost/boost-1.73.0/LICENSE_1_0.txt` manually and put this file into _vcpkg_root_path/downloads/_ directory.
Then try to rebuild this.

I can install it successfully on my local. This should be related with your local network.

```
PS F:\12011\vcpkg> .\vcpkg.exe install boost-compatibility
Computing installation plan...
The following packages will be built and installed:
    boost-compatibility[core]:x86-windows
  * boost-vcpkg-helpers[core]:x86-windows
Additional packages (*) will be modified to complete this operation.
Starting package 1/2: boost-vcpkg-helpers:x86-windows
Building package boost-vcpkg-helpers[core]:x86-windows...
-- Performing post-build validation
-- Performing post-build validation done
Building package boost-vcpkg-helpers[core]:x86-windows... done
Installing package boost-vcpkg-helpers[core]:x86-windows...
Installing package boost-vcpkg-helpers[core]:x86-windows... done
Elapsed time for package boost-vcpkg-helpers:x86-windows: 11.5 s
Starting package 2/2: boost-compatibility:x86-windows
Building package boost-compatibility[core]:x86-windows...
-- Downloading https://github.com/boostorg/compatibility/archive/boost-1.73.0.tar.gz...
-- Extracting source F:/12011/vcpkg/downloads/boostorg-compatibility-boost-1.73.0.tar.gz
-- Using source at F:/12011/vcpkg/buildtrees/boost-compatibility/src/ost-1.73.0-44642da97c
-- Packaging headers
-- Packaging headers done
-- Downloading https://raw.githubusercontent.com/boostorg/boost/boost-1.73.0/LICENSE_1_0.txt...
-- Installing: F:/12011/vcpkg/packages/boost-compatibility_x86-windows/share/boost-compatibility/copyright
-- Performing post-build validation
-- Performing post-build validation done
Building package boost-compatibility[core]:x86-windows... done
Installing package boost-compatibility[core]:x86-windows...
Installing package boost-compatibility[core]:x86-windows... done
Elapsed time for package boost-compatibility:x86-windows: 2.686 s

Total elapsed time: 14.35 s
```

**Comment by lbartoletti** [comment]
vcpkg failed also for me but on boost-regex (fresh Windows installation and first time with vcpkg):

```
.\vcpkg install cgal
[…]
Starting package 35/91: boost-regex:x86-windows
Building package boost-regex[core]:x86-windows...
-- Downloading https://github.com/boostorg/regex/archive/boost-1.73.0.tar.gz...
-- Extracting source C:/vcpkg/downloads/boostorg-regex-boost-1.73.0.tar.gz
-- Using source at C:/vcpkg/buildtrees/boost-regex/src/ost-1.73.0-a7219fe4ec
-- Including C:/vcpkg/ports/boost-regex/b2-options.cmake
-- Building x86-windows-rel
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:72 (message):
    Command failed: C:/vcpkg/installed/x86-windows/tools/boost-build/b2.exe --stagedir=C:/vcpkg/buildtrees/boost-regex/x86-windows-rel/stage --build-dir=C:/vcpkg/buildtrees/boost-regex/x86-windows-rel --user-config=C:/vcpkg/buildtrees/boost-regex/x86-windows-rel/user-config.jam -j2 --debug-configuration --debug-building --debug-generators --disable-icu --ignore-site-config --hash -q -sZLIB_INCLUDE=C:/vcpkg/installed/x86-windows/include -sBZIP2_INCLUDE=C:/vcpkg/installed/x86-windows/include -sLZMA_INCLUDE=C:/vcpkg/installed/x86-windows/include -sZSTD_INCLUDE=C:/vcpkg/installed/x86-windows/include threading=multi threadapi=win32 runtime-link=shared link=shared address-model=32 architecture=x86 toolset=msvc -sZLIB_BINARY=zlib -sZLIB_LIBPATH=C:/vcpkg/installed/x86-windows/lib -sBZIP2_BINARY=bz2 -sBZIP2_LIBPATH=C:/vcpkg/installed/x86-windows/lib -sLZMA_BINARY=lzma -sLZMA_LIBPATH=C:/vcpkg/installed/x86-windows/lib -sZSTD_BINARY=zstd -sZSTD_LIBPATH=C:/vcpkg/installed/x86-windows/lib variant=release debug-symbols=on
    Working Directory: C:/vcpkg/buildtrees/boost-regex/src/ost-1.73.0-a7219fe4ec/build
    Error code: 1
    See logs for more information:
      C:\vcpkg\buildtrees\boost-regex\build-x86-windows-rel-out.log

Call Stack (most recent call first):
  installed/x86-windows/share/boost-build/boost-modular-build.cmake:297 (vcpkg_execute_required_process)
  ports/boost-regex/portfile.cmake:12 (boost_modular_build)
  scripts/ports.cmake:73 (include)


Error: Building package boost-regex:x86-windows failed with: BUILD_FAILED
Please ensure you're using the latest portfiles with `.\vcpkg update`, then
submit an issue at https://github.com/Microsoft/vcpkg/issues including:
  Package: boost-regex:x86-windows
  Vcpkg version: 2020.02.04-nohash

Additionally, attach any relevant sections from the log files above.
PS C:\vcpkg> .\vcpkg update
Using local portfile versions. To update the local portfiles, use `git pull`.
No packages need updating.
PS C:\vcpkg> git pull
Already up to date.
```

**Comment by chatyan** [comment]
D:\Work 2019\GIT Code\vcpkg>.\vcpkg install pcl
Computing installation plan...
The following packages will be built and installed:
  * boost-asio[core]:x86-windows
  * boost-bimap[core]:x86-windows
  * boost-circular-buffer[core]:x86-windows
  * boost-context[core]:x86-windows
  * boost-coroutine[core]:x86-windows
  * boost-date-time[core]:x86-windows
  * boost-dynamic-bitset[core]:x86-windows
  * boost-endian[core]:x86-windows
  * boost-filesystem[core]:x86-windows
  * boost-foreach[core]:x86-windows
  * boost-format[core]:x86-windows
  * boost-graph[core]:x86-windows
  * boost-interprocess[core]:x86-windows
  * boost-iostreams[core]:x86-windows
  * boost-locale[core]:x86-windows
  * boost-math[core]:x86-windows
  * boost-multi-array[core]:x86-windows
  * boost-multi-index[core]:x86-windows
  * boost-multiprecision[core]:x86-windows
  * boost-parameter[core]:x86-windows
  * boost-phoenix[core]:x86-windows
  * boost-property-map[core]:x86-windows
  * boost-property-tree[core]:x86-windows
  * boost-proto[core]:x86-windows
  * boost-ptr-container[core]:x86-windows
  * boost-random[core]:x86-windows
  * boost-serialization[core]:x86-windows
  * boost-signals2[core]:x86-windows
  * boost-spirit[core]:x86-windows
  * boost-test[core]:x86-windows
  * boost-thread[core]:x86-windows
  * boost-tokenizer[core]:x86-windows
  * boost-tti[core]:x86-windows
  * boost-uuid[core]:x86-windows
  * boost-variant[core]:x86-windows
  * boost-xpressive[core]:x86-windows
  * bzip2[core]:x86-windows
  * curl[core,non-http,ssl,sspi,winssl]:x86-windows
  * double-conversion[core]:x86-windows
  * expat[core]:x86-windows
  * flann[core]:x86-windows
  * freetype[bzip2,core,png]:x86-windows
  * glew[core]:x86-windows
  * hdf5[core,szip,zlib]:x86-windows
  * jsoncpp[core]:x86-windows
  * libharu[core,notiffsymbols]:x86-windows
  * libiconv[core]:x86-windows
  * libjpeg-turbo[core]:x86-windows
  * liblzma[core]:x86-windows
  * libogg[core]:x86-windows
  * libpng[core]:x86-windows
  * libtheora[core]:x86-windows
  * libxml2[core]:x86-windows
  * lz4[core]:x86-windows
  * netcdf-c[core]:x86-windows
  * openssl[core]:x86-windows
  * openssl-windows[core]:x86-windows
    pcl[core]:x86-windows
  * pegtl-2[core]:x86-windows
  * proj4[core,database]:x86-windows
  * pugixml[core]:x86-windows
  * qhull[core]:x86-windows
  * sqlite3[core,tool]:x86-windows
  * szip[core]:x86-windows
  * tiff[core]:x86-windows
  * utfcpp[core]:x86-windows
  * vtk[core]:x86-windows
  * xxhash[core]:x86-windows
  * zlib[core]:x86-windows
  * zstd[core]:x86-windows
Additional packages (*) will be modified to complete this operation.
Starting package 1/70: boost-math:x86-windows
Building package boost-math[core]:x86-windows...
-- Downloading https://github.com/boostorg/math/archive/boost-1.73.0.tar.gz...
-- Downloading https://github.com/boostorg/math/archive/boost-1.73.0.tar.gz... Failed. Status: 18;"Transferred a partial file"
CMake Error at scripts/cmake/vcpkg_download_distfile.cmake:175 (message):

      Failed to download file.
      If you use a proxy, please set the HTTPS_PROXY and HTTP_PROXY environment
      variables to "https://user:password@your-proxy-ip-address:port/".
      Otherwise, please submit an issue at https://github.com/Microsoft/vcpkg/issues

Call Stack (most recent call first):
  scripts/cmake/vcpkg_from_github.cmake:132 (vcpkg_download_distfile)
  ports/boost-math/portfile.cmake:3 (vcpkg_from_github)
  scripts/ports.cmake:76 (include)


Error: Building package boost-math:x86-windows failed with: BUILD_FAILED
Please ensure you're using the latest portfiles with `.\vcpkg update`, then
submit an issue at https://github.com/Microsoft/vcpkg/issues including:
  Package: boost-math:x86-windows
  Vcpkg version: 2020.02.04-nohash

Additionally, attach any relevant sections from the log files above.

**Comment by chatyan** [comment]
I changed the network, it seems OK for boost installation. 
But, for installing PCL. 
There comes 
"
cl : Command line warning D9002 : ignoring unknown option '-fext-numeric-literals'
" problem

Code\vcpkg\buildtrees\pcl\src\pcl-1.9.1-f2ec88a858\pcap\include" -I"D:\Work 2019\GIT Code\vcpkg\buildtrees\pcl\src\pcl-1.9.1-f2ec88a858\png\include" -I"D:\Work 2019\GIT Code\vcpkg\buildtrees\pcl\src\pcl-1.9.1-f2ec88a858\vtk\include" -I"D:\Work 2019\GIT Code\vcpkg\buildtrees\pcl\src\pcl-1.9.1-f2ec88a858\io\include" -I"D:\Work 2019\GIT Code\vcpkg\installed\x86-windows\include\eigen3" -I"D:\Work 2019\GIT Code\vcpkg\installed\x86-windows\include" /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /FS -openmp -DBOOST_ALL_DYN_LINK -DBOOST_ALL_NO_LIB /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1    -DBOOST_ALL_NO_LIB -D_SCL_SECURE_NO_WARNINGS -D_CRT_SECURE_NO_WARNINGS -DNOMINMAX -DPCL_ONLY_CORE_POINT_TYPES /bigobj  -fext-numeric-literals /showIncludes /Foio\CMakeFiles\pcl_io.dir\src\file_io.cpp.obj /Fdio\CMakeFiles\pcl_io.dir\ /FS -c "D:\Work 2019\GIT Code\vcpkg\buildtrees\pcl\src\pcl-1.9.1-f2ec88a858\io\src\file_io.cpp"
cl : Command line warning D9002 : ignoring unknown option '-fext-numeric-literals'
ninja: build stopped: subcommand failed.

**Comment by chatyan** [comment]
Finally, it's done!

Previously I used VS2019, after I switch to Visual Studio 2015, it's done for installing PCL.

For installing boost, I changed my network.

Installing: D:/Work 2019/GIT Code/vcpkg/packages/pcl_x86-windows/share/pcl/copyright
-- Performing post-build validation
-- Performing post-build validation done
Building package pcl[core]:x86-windows... done
Installing package pcl[core]:x86-windows...
Installing package pcl[core]:x86-windows... done
Elapsed time for package pcl:x86-windows: 1.316 h
